terraform-infra-approvals/send-letter-service.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources": [
+    {"type": "azurerm_role_assignment"}
```

Comment:
jenkins doesn't have the required access to do this currently so it won't work

Reply:
😭